src/change_processor.rs

```diff
@@ -244,7 +245,45 @@ impl JobThreadContext {
                                 );
                             }
                         } else {
-                            log::warn!("Cannot change properties besides BaseScript.Source.");
+                            if instance.metadata().relevant_paths.len() >= 2 {
```

Comment:
I personally not sure if this is good enough if we where to add a new path to relevant_paths for some reason this would break 

solution would be to add a special field to metadata for meta files. still this would make this change a lot lager.